Asymmetric sync (#33)
mongodb/docs-app-services: source/sync/data-access-patterns/flexible-sync.txt

      To create queries from the .NET Client SDK, see the :ref:`.NET SDK 
      guide to Flexible Sync <dotnet-flexible-sync>`.

> Optimize Sync with Asymmetric Sync

Reviewer: ```suggestion
Optimize Sync with Asymmetric Sync (Optional)
```

Just to be clear to Flexible Sync users that they don't have to use Asymmetric Sync

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Asymmetric Sync is a sync strategy that is optimized for applications 
with heavy client-side *insert-only* workloads. You enable it for one or more 
collections. It supports writing to any collection type, including an 
{+atlas-short+} time-series collection. 

.. note::

   Asymmetric Sync is a feature of Flexible Sync and cannot be enabled on apps 
   that use Partition-Based Sync.

For example, an IoT app that frequently logs sensor data has a significant 

> write workload, and essentially no read workload. The device may also be offline 

Reviewer: ```suggestion
write workload, and no read workload. The device may also be offline 
```
let's get rid of the word essentially here, since asymmetric sync doesn't support querying.

      To create queries from the .NET Client SDK, see the :ref:`.NET SDK 
      guide to Flexible Sync <dotnet-flexible-sync>`.

> Optimize Sync with Asymmetric Sync

Author: great points!

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Asymmetric Sync is a sync strategy that is optimized for applications 
with heavy client-side write workloads. You enable it for one or more collections. 
It supports any collection type, including an {+atlas-short+} time-series 
collection.

For example, an IoT app that frequently logs sensor data has a significant 
write workload, and essentially no read workload. The device may also be offline 

> for extended periods of time. Asymmetric sync bypasses some of the processing 

Reviewer: ```suggestion
for extended periods of time. Asymmetric Sync bypasses some of the processing 
```

> required for bi-directional sync, significantly improving writes to a 

Reviewer: ```suggestion
required for bi-directional sync, significantly improving write speed to a 
```

{+atlas-short+} collection. 

> Other use cases include writing non-mutable data, such as invoices from as retail 

Reviewer: ```suggestion
Other use cases include writing immutable data, such as invoices from a retail 
```

app, or logging application events, neither of which requires conflict 
resolution.

After you have enabled Asymmetric Sync, you implement it in the client app 

> via the JavaScript SDK.

Reviewer: ```suggestion
via the client SDKs.
```

Currently, it is available in the JS SDK - it will be available in the others at some point.
